config/environments/production.rb

```diff
@@ -83,7 +83,7 @@
 
   # Configuration to stop Cost of Living landing page showing in production before
   # campaign goes live
-  if (ENV["BASIC_AUTH_USERNAME"] && ENV["BASIC_AUTH_PASSWORD"] && ENV["HEROKU_APP_NAME"]) || ENV["JENKINS"] || ENV["PUBLISHING_E2E_TESTS_COMMAND"]
+  if (ENV["BASIC_AUTH_USERNAME"] && ENV["BASIC_AUTH_PASSWORD"] && ENV["HEROKU_APP_NAME"]) || (ENV["BASIC_AUTH_USERNAME"] && ENV["BASIC_AUTH_PASSWORD"] && ENV["ENVIRONMENT"] == "integration") || ENV["JENKINS"] || ENV["PUBLISHING_E2E_TESTS_COMMAND"]
```

Comment:
Have you considered setting an environment variable in the environment to show the page?
e.g. https://github.com/alphagov/govuk-puppet/blob/98d30fd7b7dc3e9c3c50e178c3aa57c24a5cf3dc/hieradata_aws/integration.yaml#L36

You could leave `unreleased_features_enabled` set to true in integration permanently so your changes would always show up there.

We've done something similar in collections publisher: https://github.com/alphagov/collections-publisher/search?q=unreleased+features

Reply:
Nice! I'll rework - thanks!

Reply:
@leenagupte have reworked this PR now. Thanks!